### DISCUSSION TITLE
Can you sort by nulls first/last

### DISCUSSION THREAD

**Discussion by brettsaunders21** [perspective-dev/perspective#2639] [Q&A]
I see there are many different sort types, but if my column contains "nulls" I can't see any way to change sorting of nulls to be shown first or last?

**Comment by texodus** [comment]
You cannot do this generally in Perspective `2.10.1`. In the limited case where you want to just sort a flat (e.g. no `group_by` or `split_by` conditions), you can create an [expression column](https://perspective.finos.org/docs/expressions/) that maps `null` to a very large value and use this as sort, but this will not work for aggregated values (which are calculated post-expression).
